render/container.go

```diff
@@ -177,20 +169,9 @@ var ContainerImageRenderer = Memoise(FilterEmpty(report.Container,
 //
 // not memoised
 var ContainerHostnameRenderer = FilterEmpty(report.Container,
-	MakeReduce(
-		MakeMap(
-			MapContainer2Hostname,
-			ContainerWithImageNameRenderer,
-		),
-		// Grab *all* the hostnames, so we can count the number which were empty
-		// for accurate stats.
```

Comment:
Where would these stats be displayed?

Reply:
Bottom-left of the screen, like "21 nodes (119 filtered)"

Reply:
Ok cool. That information's been misleading and incorrect at times for container topologies so I'm ok seeing it removed.

Looking at the UI code, it seems it's already robust enough to not display filtered nodes count if it's not provided so probably nothing will need to be updated there :+1: